apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,31 @@
+import { IDetailQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import './transcribed-audio-detail.full-view.presenter.css';
+
+export const TranscribedAudioDetailFullViewPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
+
+    return (
+        <div
+            className="detail-full-view-container"
+            title="View Connected Photograph"
+            data-testid={transcribedAudioItem.id}
+        >
+            <div className="detail-full-view-media-container">
+                <MediaPlayer audioUrl={transcribedAudioItem.audioURL} />
+            </div>
+            <div className="detail-full-view-meta-container">
+                <strong>Transcribed Audio ID:</strong> {transcribedAudioItem.id}
+                <br />
+                <strong>Run Time:</strong> {runTime} Sec
+                <br />
+                <strong>Audio Url:</strong> {transcribedAudioItem.audioURL}
+                <br />
+                <strong>Transcription:</strong> {transcribedAudioItem.plainText}
```

Comment:
I'd have a custom component for displaying the transcript. This is because in the future we will change the behaviour when we introduce more intricate views of text data (3 way translation, etc.).

Reply:
Should I leave this for now?  As we talked about, ideally the view model would have a nested transcription object that has the in and out point for the player to use.  I started to get into parsing the `plainText` string with regexp to get that info out but realized that it would be thrown away once the model is updated.

Reply:
Added `SinglePropertyPresenter` to the utils dir and added this to the `full-view` and `thumbnail`

Reply:
Also, I had to use a template string in the `value` for the duration to be able to have the "sec" unit come after the number of seconds.